send.js

```diff
@@ -27,10 +28,30 @@ const image = function(channel, url) {
     .catch(console.error);
 };
 
+const reaction = function(channel, messageTimestamp, emoji) {
+  return web.reactions
+    .add({
+      channel,
+      timestamp: messageTimestamp,
+      name: emoji
+    })
+    .catch(console.error);
+};
+
 module.exports.confirmation = {
-  taco: function(channel, userFrom, userTo) {
-    const text = `<@${userTo}> received a :taco: from <@${userFrom}>`;
-    return message(channel, text);
+  emojis: _.shuffle([
+    'ok_hand',
+    'fire',
+    'thumbsup',
+    'boom',
+    'tada',
+    'heart_eyes'
+  ]),
+  index: 0,
+  reaction(channel, messageTimestamp) {
```

Comment:
instead of having to shuffle and then keep of closure of the index, why not just pick a random number.
```
const randomIndex = Math.floor((Math.random()* 6)+1);
const emoji = this.emojis[randomIndex];
```

Reply:
Because if you give 5 tacos in one message I want to send 5 different reactions, since I can't react with the same emoji multiple times.